pkg/autoscaler/autoscaler_server.go

```diff
@@ -40,6 +40,7 @@ type AutoScaler struct {
 	clock         clock.Clock
 	stopCh        chan struct{}
 	readyCh       chan<- struct{} // For testing.
+	health        *healthInfo
```

Comment:
realizing this isn't the best naming for what it's doing. will rename after i get some feedback